### ISSUE TITLE
Note's header link

### ISSUE THREAD

**Issue by fxsalazar** [lumen-notes/lumen#170]
What if: `[[<note-id>#<Header>|<text-to-show>]]`

**Comment by colebemis** [comment]
Yeah, this seems reasonable. Looks like Obsidian supports this syntax too: https://help.obsidian.md/Linking+notes+and+files/Internal+links#Link+to+a+heading+in+a+note.

Just curious, what's your use case? Why would this be helpful for you?

**Comment by fxsalazar** [comment]
I have a note with multiple lists and wanted to reference a list in a item of another list; I also realized that I already have notes with heading references, from Obsidian times I guess.

Theses notes aren't that _atomic_, if it is this you are wondering, I could split the main note and use normal links, or embed notes; but sometimes thoughts are long and before giving it a structured in separate notes, I use headers to split them and thing let to another (headers linking); another use case is when making compilations of things, even though the note might be temporal (or not) it's nice to link it with things that already exists.
